src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -149,3 +154,11 @@
     {% if form is not rootform %}</span>{% else %}</div>{% endif %}
     {%- endif %}
 {%- endblock form_errors %}
+
+{# Help #}
+
+{% block form_help -%}
+    {%- if help is not empty -%}
+        <span id="{{ id }}_help" class="help-block">{{ translation_domain is same as(false) ? help : help|trans({}, translation_domain) }}</span>
```

Comment:
We shouldn't use the ternary operator here (see #25744).

Reply:
same in the other places

Reply:
Ok, I fix it too.
Just a general question for my general culture, why it works with if/else and not with a ternary ? (For me it's the same)